utils/text_formatting.jsx

```diff
@@ -180,7 +182,12 @@ export function autolinkAtMentions(text, tokens) {
     }
 
     let output = text;
-    output = output.replace(/\B@([a-z0-9.\-_]*)/gi, replaceAtMentionWithToken);
+
+    let match = output.match(AT_MENTION_PATTERN);
+    while (match && match.length > 0) {
```

Comment:
`AT_MENTION_PATTERN` will match just `@` without any actual username. Is that expected?

Reply:
Yes, that's expected.  Even without corresponding user, it will be tagged with a token `$MM_ATMENTION0$` and then will convert into corresponding HTML `<p><span data-mention="foo.com">@foo.com</span></p>`.  Such HTML will render as is by `AtMention` component.